### ISSUE TITLE
Still using self-signed SSL

### ISSUE THREAD

**Issue by PureStrgUser01** [PureStorage-OpenConnect/pure1-unplugged#9]
Hello,

Testing "Pure1-unplugged" for several weeks, all is fine except that I can't make it to use my own SSL certificates.

I started with
`createSelfSignedCerts: true`
in my /etc/pure1-unplugged/config.yaml

But now, even with :
`createSelfSignedCerts: false`
"Pure1-unplugged" never use my customized key and cert files :
```
sslCertFile: /etc/pure1-unplugged/ssl/myhost1.dev1.cea.fr.pem.crt
sslKeyFile: /etc/pure1-unplugged/ssl/myhost1.dev1.cea.fr.key
```

It looks like if it was using:
/var/lib/docker/overlay2/c22d8964391591b87defffc4bd778cdab2b501bd887e05534371ac2bf90f6758/merged/etc/ingress-controller/ssl/default-fake-certificate.pem

and not:
/var/lib/docker/overlay2/c22d8964391591b87defffc4bd778cdab2b501bd887e05534371ac2bf90f6758/merged/etc/ingress-controller/ssl/pure1-unplugged-pure1-unplugged-https-cert.pem

I have re-compliled (how do you call that ?) with these commands:
```
puctl infra reset ; puctl infra init ; puctl infra status
puctl pure1-unplugged install ; puctl pure1-unplugged status
```

and even reboot my Linux server...

Any ideas ? Bug ?

Best regards.

**Comment by Pure-AdamuKaapan** [comment]
@PureStrgUser01 it does in fact look like you've found a bug with our SSL cert handling. I managed to reproduce it locally and found some quick steps to theoretically fix it. I'll keep posting updates here and a fix should hopefully be out in the next release we cut.

**Comment by Pure-AdamuKaapan** [comment]
Hello, just wanted to leave a quick update here: I still believe I've found the issue seen here, we've been having some build infrastructure problems on our side that I believe we've just hopefully solved (see #16). I'll continue to post updates here as they come. Thanks for your patience!

**Comment by Pure-AdamuKaapan** [comment]
Good news! It worked 😄 #16 has been closed, and I've also opened #19 just to make sure the SSL secret gets cleaned up on an uninstall.
